kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/db/keyvalue/KeyValueDbService.java

```diff
@@ -0,0 +1,112 @@
+package org.eclipse.kura.db.keyvalue;
+
+/*******************************************************************************
+ * Copyright (c) 2023 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+
+import org.eclipse.kura.KuraException;
+import org.eclipse.kura.connection.listener.ConnectionListener;
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * {@link KeyValueDbService} provides APIs to use the functionalities a of a KeyValue database.
```

Comment:
Maybe we can explain better what we mean with KeyValue database.. something like this?
```suggestion
 * {@link KeyValueDbService} provides APIs to use the functionalities a of a database capable of storing data in form of key-value pairs.
```